http-clients/aws-crt-client/src/main/java/software/amazon/awssdk/http/crt/AwsCrtAsyncHttpClient.java

```diff
@@ -264,36 +281,50 @@ public void close() {
      */
     public interface Builder extends SdkAsyncHttpClient.Builder<AwsCrtAsyncHttpClient.Builder> {
 
-        /**
-         * The number of Threads to use in the EventLoop.
-         * @param eventLoopSize The number of Threads to use in the EventLoop.
-         * @return the builder of the method chaining.
-         */
-        Builder eventLoopSize(int eventLoopSize);
-
         /**
          * The AWS CRT TlsCipherPreference to use for this Client
          * @param tlsCipherPreference The AWS Common Runtime TlsCipherPreference
-         * @return the builder of the method chaining.
+         * @return The builder of the method chaining.
          */
         Builder tlsCipherPreference(TlsCipherPreference tlsCipherPreference);
 
         /**
          * Whether or not to Verify the Peer's TLS Certificate Chain.
          * @param verifyPeer true if the Certificate Chain should be validated, false if validation should be skipped.
-         * @return the builder of the method chaining.
+         * @return The builder of the method chaining.
          */
         Builder verifyPeer(boolean verifyPeer);
 
+        /**
+         * If set to true, then the TCP read back pressure mechanism will be enabled.
```

Comment:
make sure to call out if this is ever set to true the user is responsible for calling updateWindow on the stream object.

Reply:
sorry, incrementWindow()

Reply:
I'll update the comment!